package.json

```diff
@@ -17,13 +19,23 @@
   },
   "homepage": "https://github.com/anastasiya-nd/budget#readme",
   "devDependencies": {
+    "@babel/core": "^7.15.0",
+    "@babel/preset-env": "^7.15.0",
+    "@babel/preset-react": "^7.14.5",
+    "babel-loader": "^8.2.2",
+    "css-loader": "^6.2.0",
     "editor": "^1.0.0",
     "eslint": "^7.32.0",
     "eslint-config-standard": "^16.0.3",
```

Comment:
Let's use `airbnb` code style instead of `standard`

Reply:
I have to remove eslint and install to main